### DISCUSSION TITLE
How to move a window to a different monitor?

### DISCUSSION THREAD

**Discussion by hopezh** [mrkai77/Loop#544] [Q&A] [answered]
Loop is elegant and intuitive, and I love it! 

However, I haven't find a way, or the setting, to **move a window to a different monitor**, for multi-monitor set up. 

Appreciate your help.

**Comment by SenpaiHunters** [comment, marked as answer]
![000954-08@2x](https://github.com/user-attachments/assets/a0903105-3b8c-43a5-b233-aaef4f33cdde)

**Reply by hopezh** [reply]
Works like a charm. Thanks.
